rules/scripts/bitstreams_manifest.schema.json

```diff
@@ -6,88 +6,78 @@
       "A manifest of bitstreams in a cache entry, informing what is there and how to reproduce",
   "type": "object",
   "properties": {
-    "schemaVersion": {
+    "schema_version": {
       "description": "Version number of this manifest's schema",
       "type": "number",
       "minimum": 1,
       "maximum": 1
     },
-    "buildId": {
+    "build_id": {
       "description": "Build ID associated with this entry (typically a git hash)",
       "type": "string"
     },
-    "outputFiles": {
-      "description": "Map of output file paths to their metadata objects",
+    "designs": {
+      "description": "Map of designs to their files and metadata objects",
       "type": "object",
-      "additionalProperties": { "$ref": "#/$defs/outputFile" }
+      "additionalProperties": { "$ref": "#/$defs/design" }
     }
   },
   "required": [
-    "schemaVersion",
-    "buildId",
-    "outputFiles"
+    "schema_version",
+    "designs"
   ],
   "$defs": {
-    "bitstreamInfo": {
+    "bitstream_info": {
       "description": "Bitstream build output",
       "type": "object",
       "properties": {
-        "@type": {
-          "description": "Tag for type of this object",
-          "type": "string",
-          "const": "bitstreamInfo"
+        "file": {
+          "description": "Path to bitstream file relative to root",
+          "type": "string"
         },
-        "design": {
-          "description": "Design name / top for the bitstream",
+        "build_target": {
+          "description": "Build target that generated the output",
           "type": "string"
         }
       },
       "required": [
-        "@type",
-        "design"
+        "file"
       ]
     },
-    "memoryMapInfo": {
+    "memory_map_info": {
       "description": "Build output that maps memories to cells in a hardware implementation",
       "type": "object",
       "properties": {
-        "@type": {
-          "description": "Tag for type of this object",
-          "type": "string",
-          "const": "memoryMapInfo"
-        },
-        "design": {
-          "description": "Design name / top for the bitstream",
+        "file": {
+          "description": "Path to file with memory contents, relative to root",
           "type": "string"
         },
-        "memoryId": {
-          "description": "Name or key identifying the associated memory",
+        "build_target": {
+          "description": "Build target that generated the output",
           "type": "string"
         }
       },
       "required": [
-        "@type",
-        "design",
-        "memoryId"
+        "file"
       ]
     },
-    "outputFile": {
-      "description": "Information about a build output file",
+    "design": {
+      "description": "Files and metadata related to a design",
       "type": "object",
       "properties": {
-        "buildTarget": {
-          "description": "Build target that generated the output",
+        "build_id": {
```

Comment:
Why is there a `build_id` per design in addition to the `build_id` of the manifest?

Reply:
Per design is to track where the design files actually originate (sha for the built commit). The per-manifest part is kind of superfluous, but it tracks when the collection was created. These are simply meant to be informative for debugging.

Reply:
Okay, if we already think that the per-manifest ID will be superfluous, we may want to omit it. It could lead to mistakes when it differs from the per-build ID.

Reply:
Removed.